dash_table/package-info.json

```diff
@@ -23,8 +23,8 @@
     "private::wait_dash8082": "wait-on http://localhost:8082",
     "private::wait_dash8083": "wait-on http://localhost:8083",
     "private::wait_js": "wait-on http://localhost:8080",
-    "private::lint.js": "eslint src",
-    "private::lint.ts": "tslint --project . src/**/*.ts",
+    "private::lint.js": "eslint '{src,demo,tests}/**/*.js'",
+    "private::lint.ts": "tslint '{src,demo,tests}/**/*.{ts,tsx}' --exclude '**/@Types/*.*'",
```

Comment:
Won't tslint run on .js files too? Is it necessary to have both `eslint` and `tslint`?